### DISCUSSION TITLE
access eurostat databases

### DISCUSSION THREAD

**Discussion by Krisselack** [rOpenGov/eurostat#205] [Q&A]
is it possible to directly access eurostat databases with this package? 
I wanted to access comext and prom through R. Is this possible? 

If yes, could you provide any examples? 
If no, would you know how to accomplish sthg. like that?

**Comment by antagomir** [comment]
This package has been designed for eurostat open data API but does not include tools for closed parts.

Is this what you were after?

**Reply by Krisselack** [reply]
no, you can access comext as well as prom without registration. 
# comext 
http://epp.eurostat.ec.europa.eu/newxtweb/

# prom 
https://ec.europa.eu/eurostat/web/prodcom/data/database

Would it be possible to run queries through R?

**Reply by jhuovari** [reply]
Unfortunately comext and prom data are not part of eurostat api. Comext can be download from Bulk Download facility, but not in same form as other eurostat data. So, with currently eurostat package can't be use to access those data.

**Reply by Krisselack** [reply]
Thank you for the fast answer!
